### PR TITLE
completed html-web-page-challenge

### PR DESCRIPTION
**Pull request by dcatan89**
Screen Shot 2021-11-30 at 6 01 59 PM

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey DJ! Your indentation in this document is pretty all over the place, please revise to follow proper HTML formatting rules.

**Comment by dcatan89** [comment]
Made changes to reflect better html indentation.

File: html-web-page/index.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+              <td></td>
+              <td></td>
+              <td>&check;</td>
+        </tr>
+        <tr>
+              <td>Monthly Pricing</td>
+              <td>$30</td>
+              <td>$50</td>
+              <td>$60</td>
+        </tr>
```
Comment (codingcodymiller) [inline html-web-page/index.html]: This indentation is still a total mess.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: html-web-page/index.html (hunk 1 above)
Comment (dcatan89) [reply html-web-page/index.html]: Indentation has been fixed.

**Review by codingcodymiller — APPROVED** [review]
Great work!

**Pull request merged**